apps/docs/pages/guides/integrations/prisma.mdx

```diff
@@ -252,7 +238,17 @@ model User {
 
 To learn more about using Prisma with multiple database schemas, refer to [this page in the Prisma docs](https://www.prisma.io/docs/guides/database/multi-schema#learn-more-about-the-multischema-preview-feature).
 
-### 3. Enabling PosgreSQL extensions
+### Using Supabase Row Level Security with Prisma
```

Comment:
we can change it to "Postgres Row Level Security" if you want - it's not specific to Supabase